apps/server/lib/http/routes.ts

```diff
@@ -7,6 +7,7 @@ import type {
 	Sync,
 	Worker,
 } from '@balena/jellyfish-worker';
+import { hydraAdmin } from '@balena/jellyfish-plugin-oauth/build/actions/action-sync-oauth-client';
```

Comment:
We shouldn't deep import from under `build/` if possible. I've updated `plugin-oauth` to export this function from the main index file so you should be able to `import { hydraAdmin } from '@balena/jellyfish-plugin-oauth';`

https://github.com/product-os/jellyfish-plugin-oauth/pull/6

Reply:
Thanks Josh!